include/nitro/broken_options/option/group.hpp

```diff
@@ -38,10 +38,10 @@ namespace nitro
 {
 namespace broken_options
 {
-    class group
+    class option_group
```

Comment:
That's a misnomer in nitro language. It represents a group of arguments, not options. And while you're on it, rename the options_ member as well.